cub/cub/device/dispatch/dispatch_scan.cuh

```diff
@@ -37,6 +37,8 @@
 
 #include <cub/config.cuh>
 
+#include "cub/util_namespace.cuh"
```

Comment:
important: we prefer `<>` includes throughout the project. Functionally, this is reported to lead to a better NVRTC performance. 
```suggestion
#include <cub/util_namespace.cuh>
```